### ISSUE TITLE
RFE: Add man_mande=cutline overlay

### ISSUE THREAD

**Issue by Talkless** [hholzgra/maposmatic#126]
OpenTopoMap is useful for compas & map navigation, but for some reason they do not render cutlines.

[man_mande=cutline](https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dcutline) can be used as paths while hiking, also as reference points for navigation. Our government topographical maps does show cutlines, but it's not as detailed as our OSM with all small paths...

**Comment by hholzgra** [comment]
That would be an issue for the upstream OpenTopoMap project instead:

https://github.com/der-stefan/OpenTopoMap

**Comment by Talkless** [comment]
Yeah I have issue there created too, but if the don't want to change long-lasting style, alternative would be to add lines as a separate layer here.

https://github.com/der-stefan/OpenTopoMap/issues/353
